packages/@aws-cdk/aws-apigateway/test/method.test.ts

```diff
@@ -67,7 +67,7 @@ describe('method', () => {
     new apigw.Method(stack, 'my-method', {
       httpMethod: 'POST',
       resource: api.root,
-      integration: new apigw.AwsIntegration({ service: 's3', path: 'bucket/key' }),
+      integration: new apigw.AwsIntegration({ service: 's3', path: 'bucket/key', region: 'eu-west-1' }),
```

Comment:
I would prefer if you create a separate test case (something like 'provided region is correctly configured'), rather than an update an existing test case.